--- /etc/sudoers	2025-06-17 12:02:35.292628715 -0400
+++ sudoers	2025-06-17 12:03:47.102620431 -0400
@@ -43,7 +43,7 @@
 ##
 ## Preserve editor environment variables for visudo.
 ## To preserve these for all commands, remove the "!visudo" qualifier.
-Defaults!/usr/bin/visudo env_keep += "SUDO_EDITOR EDITOR VISUAL"
+# Defaults!/usr/bin/visudo env_keep += "SUDO_EDITOR EDITOR VISUAL"
 ##
 ## Use a hard-coded PATH instead of the user's to find commands.
 ## This also helps prevent poorly written scripts from running
@@ -122,7 +122,7 @@
 root ALL=(ALL:ALL) ALL
 
 ## Uncomment to allow members of group wheel to execute any command
-# %wheel ALL=(ALL:ALL) ALL
+%wheel ALL=(ALL:ALL) ALL
 
 ## Same thing without a password
 # %wheel ALL=(ALL:ALL) NOPASSWD: ALL
